File: crates/deltoids/tests/diff_cases/cases/231-ruby-rspec-multiple-new-tests-in-replace/3-updated.rb
RSpec.describe Client do
  it "recovers via the slow path" do
    setup_stub
    expect(result).to eq(github_id: 1, login: "ghost")
  end

  it "raises when recovery returns nothing" do
    setup_stub
    expect { run }.to raise_error(RecoveryError)
  end

  it "retries on transient errors" do
    setup_stub
    expect(client.retries).to eq(3)
  end

  it "logs the recovery attempt" do
    expect(logger).to have_received(:info)
  end

  it "exposes the total count" do
    expect(client.total).to eq(1)
  end
end

File: crates/deltoids/tests/diff_cases/cases/233-go-subtest-new-in-replace/3-updated.go
package client

import "testing"

func TestClient(t *testing.T) {
	t.Run("recovers via the slow path", func(t *testing.T) {
		setupStub()
		if got := result(); got != "ghost" {
			t.Fatalf("got %q", got)
		}
	})

	t.Run("raises when recovery returns nothing", func(t *testing.T) {
		setupStub()
		if err := run(); err == nil {
			t.Fatal("expected error")
		}
	})

	t.Run("exposes the total count", func(t *testing.T) {
		if got := total(); got != 1 {
			t.Fatalf("got %d", got)
		}
	})
}
